finger-exercises-react/react/ecmascript/01 - spread/index.js

```diff
@@ -1,8 +1,12 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
```

Comment:
Ok. I'm on it. I did it like this because in the third test I have the following: 

`expect(min(1, 2, 3, 4, 5, 0)).toBe(0);`

So I thought I could get that list of arguments as an array with the spread operator since the number of arguments is not always the same. Shouldn't I take every test into consideration ?

Reply:
Ohh okey, if that's the case then it's all right. I thought you would pass the array as a list into the function